src/twisted/internet/iocpreactor/iocpsupport.py

```diff
@@ -0,0 +1,20 @@
+# -*- test-case-name: twisted.internet.test.test_iocp -*-
+# Copyright (c) Twisted Matrix Laboratories.
+# See LICENSE for details.
+
+"""
+A shim to the underlying IOCP support code.
+
+See U{https://github.com/twisted/twisted-platform-support}.
+"""
+
+from twisted_platform_support._iocp import (
```

Comment:
why are we importing private stuff from an external library?

if those attributes in twisted_platform_support are supposed to be used outside of twisted_platform_support, I think that they should be public.

If the whole twisted_platform_support is private to twisted, then I think that we should keep it in the main twisted repo and make it easy to create new versions and run tests.

with an external library, we would either need to duplicate a lot of tests... or release minor version each need we need to test some change